myosuite/tests/test_OSL.py

```diff
@@ -0,0 +1,34 @@
+import gymnasium as gym
+import myosuite
+from mujoco import viewer
+import time
+
+
+def run_env():
```

Comment:
Check how other envs are being tested. It's much more comprehensive.

Reply:
This was a duplicate test. Removed it